src/utils.cc

```diff
@@ -152,15 +152,15 @@ bool Utils::SaveFile(const char *Path, char *Buf, long L)
 
 /*** Met le bon chemin pour charger un fichier ***/
 /*************************************************/
-#if (defined(__unix__) || defined(ANDROID)) && !defined(__AMIGAOS4__)
+#if (defined(__unix__) || defined(ANDROID)) || defined(__HAIKU__) && !defined(__AMIGAOS4__)
 // Version Linux
 void Utils::GetPath(char *Path)
 {
     char Provi[512];
 
     strcpy(Provi, Path);
 
-#ifndef ANDROID
+#if !defined(ANDROID) && !defined(__HAIKU__)
```

Comment:
it should not be needed here too, this `if` checks the path passed as argument if there is one, which is not our usecase, so it should "jump" over the next checks

Reply:
you're right, this path is only defined for __unix__: https://github.com/search?q=repo%3Apetitlapin%2FLi-Ri%20defpath&type=code. It is mostly used for development purpose (to specify a path when testing), so keeping as you did is good enough for me

Reply:
yes, it also needs to be added in the main.cc wherever there is a #ifdef __unix__ for this variable:
https://github.com/petitlapin/Li-Ri/blob/041556ceefe1815ef84109faaff1b0c3cc4d87db/src/main.cc#L65
https://github.com/petitlapin/Li-Ri/blob/041556ceefe1815ef84109faaff1b0c3cc4d87db/src/main.cc#L73
https://github.com/petitlapin/Li-Ri/blob/041556ceefe1815ef84109faaff1b0c3cc4d87db/src/main.cc#L94